### ISSUE TITLE
arm build please?

### ISSUE THREAD

**Issue by qbert911** [cburgdorf/rusty-rlp#17]
Raspberry Pi can't use :(

**Comment by marioevz** [comment]
web3py install fails in raspberry pi due to this package not being available in Raspberry Pi.

**Comment by cburgdorf** [comment]
I'm afraid I don't know how to support ARM builds right now. I build binaries through GitHub Actions. If anyone knows how to add Arm support I would be happy to merge such a PR.

This would be the place to start looking into I guess

https://github.com/cburgdorf/rusty-rlp/blob/41af02048f60a8ce4f4193a03c4b8f3a04b4491f/.github/workflows/main.yml#L44-L46

In the meantime, there is work under way to fall back to the slower Python based implementation for environments where this library is not available.

https://github.com/ethereum/pyrlp/pull/127

**Comment by Naheel-Azawy** [comment]
Hello @cburgdorf, I've been following up with your work on ethereum/pyrlp#127 and this is a real life saver. Thanks!
But when I try to install web3 from pip, it still gets the older version. I guess that's probably because web3 repo has not been updated for that yet. Do you suggest any quick, maybe dirty, solution for that?

**Comment by Naheel-Azawy** [comment]
@cburgdorf Thanks I really appreciate it. Will be waiting for that.

**Comment by cburgdorf** [comment]
@Naheel-Azawy Can you try again. I think the error should be gone.

**Comment by Naheel-Azawy** [comment]
@cburgdorf Yep, I do confirm that web3 py was installed on my raspberry pi zero w with no errors using pyrlp. Thank you!
I was trying to perform some actual work with it but I'm now getting a ConnectionRefusedError. It works well on my localhost though. But anyway, I doubt this is related to our issue here. What I'm trying to say is that I couldn't actually test it in a real scenario yet. Will update you once I figure out what's going wrong.

**Comment by cburgdorf** [comment]
@Naheel-Azawy Sounds good. Yes, the other error is most likely unrelated. This library is now entirely optional for py-rlp and needs to be explicitly installed via an extra (e.g. `rlp[rust-backend]`) and pretty much nothing else changed about the py-rlp from the previous version.
